pulpcore/cli/common/context.py

```diff
@@ -512,7 +512,7 @@ def update(
         if parameters:
             _parameters.update(parameters)
         return self.call(
-            "update",
+            "partial_update",
```

Comment:
This is the only change with a risk to break anyone. But i don't suspect anyone to use `update` aka `PUT` for any good reason.
(Yes our upload api uses it to create chunks. You'd kind of expect a POST call for that, no? And that weird case is covered explicitely in this PR.)
...is what i wanted to say. But thinking again, this is breaking all contexts using UPDATE_ID today and defining neither PARTIAL_UPDATE_ID or nor ID_PREFIX.

Reply:
I think that is a fine assumption to make. I looked at the other two plugins (deb & ostree) and they both use partial_update, so I think we are good.

Reply:
I added a workaround. Please look again. @ggainey

Reply:
Yes, you are right. I had to go look at #444 to understand how this works.